### PR TITLE
WIP: Added CHANGELOG.md file. Adjusted buildscript to read it directly

### PR DESCRIPTION
**Pull request by takanuva15**
Hi mtrnord,

I'm still learning my way around the repository, but I saw immediately that the changelog is being duplicated, which is unnecessary. This PR will allow us to just make changes to CHANGELOG.md file, and the build script will automatically read the changelog to update the change notes in the plugin xml.

**Please don't merge the PR yet. I want to make sure you are okay with the changes, and then I will add some additional code to this PR so that only the changelog stuff for the most recent version is added to the change notes**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import org.jetbrains.grammarkit.tasks.GenerateLexer
 import org.jetbrains.grammarkit.tasks.GenerateParser
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import com.github.rjeschke.txtmark.Processor
 
 plugins {
     idea
-    id("org.jetbrains.intellij") version "0.4.20"
+    id("org.jetbrains.intellij") version "0.4.21"
```
Comment (MTRNord) [inline build.gradle.kts:8]: Please not yet update this to the latest as for some reason the newest version introduces some bug where the plugin crashes. There is a upstream issue over at Jetbrains about it. It is only fixed in the next release of their IDE :/

Comment (takanuva15) [reply build.gradle.kts:8]: will do

Comment (MTRNord) [reply build.gradle.kts:8]: To be exact this update caused at least on my setup issue #13

**Comment by MTRNord** [comment]
Thanks for the PR :)

**Pull request merged**